crates/bitwarden-crypto/src/enc_string/symmetric.rs

```diff
@@ -268,13 +355,29 @@ impl schemars::JsonSchema for EncString {
 
 #[cfg(test)]
 mod tests {
+    use generic_array::GenericArray;
     use schemars::schema_for;
 
     use super::EncString;
     use crate::{
         derive_symmetric_key, CryptoError, KeyDecryptable, KeyEncryptable, SymmetricCryptoKey,
     };
 
+    #[test]
+    fn test_enc_roundtrip_xchacha20() {
+        let key_id = [0u8; 24];
+        let enc_key = [0u8; 32];
+        let key = SymmetricCryptoKey::XChaCha20Poly1305Key(crate::XChaCha20Poly1305Key {
+            key_id,
+            enc_key: Box::pin(*GenericArray::from_slice(enc_key.as_slice())),
```

Comment:
Similar to some of the other changes
```suggestion
            enc_key: Box::pin(enc_key.into()),
```